devlab/utils/os.sh

```diff
@@ -0,0 +1,332 @@
+#!/bin/sh
+
+cfg=devlab/config.ini
+SRC=`grep grizzly_ip $cfg | awk '{print $3}'`
+DST=`grep icehouse_ip $cfg | awk '{print $3}'`
+DST_SSH_CMD="ssh -o StrictHostKeychecking=no root@${DST}"
+SRC_SSH_CMD="ssh -o StrictHostKeychecking=no root@${SRC}"
+DST_ENV=`cat << EOF
+declare -x OS_AUTH="http://${DST}:35357/v2.0";
+declare -x OS_AUTH_URL="http://${DST}:35357/v2.0";
+declare -x OS_PASSWORD="admin";
+declare -x OS_TENANT_NAME="admin";
+declare -x OS_USERNAME="admin";
+EOF`
+SRC_ENV=`cat << EOF
+declare -x OS_AUTH="http://${SRC}:35357/v2.0";
+declare -x OS_AUTH_URL="http://${SRC}:35357/v2.0";
+declare -x OS_PASSWORD="admin";
+declare -x OS_TENANT_NAME="admin";
+declare -x OS_USERNAME="admin";
+EOF`
+DST_SSH="${DST_SSH_CMD} ${DST_ENV}"
+SRC_SSH="${SRC_SSH_CMD} ${SRC_ENV}"
+net_service="neutron"
+
+#define functions
+#arg1 - project name
+#arg2 - project description
+function CREATE_TENANT {
```

Comment:
Uppercase is usually used for variables in bash. Functions should use lower case letters with underscores. Please reformat